data-loader/cli/src/main/java/com/scalar/db/dataloader/cli/command/dataexport/ScanOrderingConverter.java

```diff
@@ -1,20 +1,23 @@
 package com.scalar.db.dataloader.cli.command.dataexport;
 
+import static com.scalar.db.dataloader.cli.util.CommandLineInputUtils.splitByDelimiter;
+
 import com.scalar.db.api.Scan;
+import com.scalar.db.common.error.CoreError;
 import java.util.ArrayList;
 import java.util.List;
 import picocli.CommandLine;
 
 public class ScanOrderingConverter implements CommandLine.ITypeConverter<List<Scan.Ordering>> {
   @Override
   public List<Scan.Ordering> convert(String value) {
-    String[] parts = value.split(",");
+    String[] parts = splitByDelimiter(value, ",", 0);
     List<Scan.Ordering> scanOrderList = new ArrayList<>();
     for (String part : parts) {
-      String[] data = part.split("=");
-
+      String[] data = splitByDelimiter(part, "=", 2);
```

Comment:
Why don't you use `parseKeyValue()`?

Reply:
@komamitsu san, 
We are reading the scan order input here, so it will be in `column=sort_order` format (sort order will be always ascending or descending ex.`asc` or 'dsc`) and in `parseKeyValue` we are reading a column name and its value. So even though we may be able to use `parseKeyValue`, it will not be correct considering the context. Please do let me know if you think it is okay to be used in this case as well.

Reply:
Ah, I think `com.scalar.db.dataloader.cli.util.CommandLineInputUtils#parseKeyValue()` should be more generic and not return `ColumnKeyValue` since it's `CommandLineInputUtils#parseKeyValue()`. I'll put a comment on the method.

Reply:
Added a comment https://github.com/scalar-labs/scalardb/pull/2617/files#r2099169502